tests/components/calendar/test_google.py

```diff
@@ -369,8 +359,7 @@ def test_all_day_offset_event(self, mock_next_event):
                 'date': tomorrow.isoformat()
             },
             'end': {
-                'date': (tomorrow + dt_util.dt.timedelta(days=1))
-                .isoformat()
+                'date': (tomorrow + dt_util.dt.timedelta(days=1)).isoformat()
```

Comment:
Try not to clean up the code unless you're actually touching it, it makes the diff of actual changes harder to read.